examples/langchain/Gemini_LangChain_QA_Chroma_WebLoad.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>There's an error saved in this notebook about a deprecated symbol. To "fix" it, it might be better to pin the version of langchain that is installed to one that doesn't use deprecated API calls. This will make sure the notebook always runs, and makes it clear to readers the version we expect (in case they are using something different)</p><p> </p>

<pre class="ql-syntax" spellcheck="false">LangChainDeprecationWarning: The method BaseRetriever.get_relevant_documents was deprecated in langchain-core 0.1.46 and will be removed in 0.3.0.
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>